modules/ticket_selector/DisplayTicketSelector.php

```diff
@@ -259,6 +259,33 @@ public function display($event = null, $view_details = false)
     protected function activeEventAndShowTicketSelector($event, $_event_active_status, $view_details)
     {
         $event_post = $this->event instanceof EE_Event ? $this->event->ID() : $event;
+        
+        d($this->display_full_ui());
+        d($this->event->display_ticket_selector());
+        d($view_details);
+        d($_event_active_status);
+        d(
+                   ! $this->event->display_ticket_selector()
+                   || $view_details
+                   || post_password_required($event_post)
+                   || (
+                       $_event_active_status !== EE_Datetime::active
+                       && $_event_active_status !== EE_Datetime::upcoming
+                       && $_event_active_status !== EE_Datetime::sold_out
+                       && ! (
+                           $_event_active_status === EE_Datetime::inactive
+                           && is_user_logged_in()
+                       )
+                   ));
+        d((
+                       $_event_active_status !== EE_Datetime::active
+                       && $_event_active_status !== EE_Datetime::upcoming
+                       && $_event_active_status !== EE_Datetime::sold_out
+                       && ! (
+                           $_event_active_status === EE_Datetime::inactive
+                           && is_user_logged_in()
+                       )));
+
```

Comment:
what's this?  is this from the Kint debugger?

Reply:
Oops! My bad.

Yeah that Kint and I completely forgot I had that code in there from when I was figuring out which conditional was 'failing'.